src/kibali/Scheme.cs

```diff
@@ -23,6 +23,7 @@ public void Write(Utf8JsonWriter writer)
             if (!string.IsNullOrEmpty(UserDescription)) writer.WriteString("userDescription", UserDescription);
             if (RequiresAdminConsent) writer.WriteBoolean("requiresAdminConsent", RequiresAdminConsent);
             if (PrivilegeLevel != 0) writer.WriteNumber("privilegeLevel", PrivilegeLevel);
+            if (IsPreauthorizationOnly) writer.WriteBoolean("isPreauthorizationOnly", IsPreauthorizationOnly);
```

Comment:
Just a suggestion, value will always be true...
```suggestion
            if (IsPreauthorizationOnly) writer.WriteBoolean("isPreauthorizationOnly", true);
```